mscore/pm.cpp

```diff
@@ -18,6 +18,11 @@
 //  Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
 //=============================================================================
 
+#include "musescore.h"
+#include "preferences.h"
+#include "pm.h"
+#include "seq.h"
+
```

Comment:
note that moving these above the #if define is necessary to get the Q_OS_WIN definition.